src/data/forecast.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Could this be removed now that we're fetching data from an API? If its now redundant it should be deleted